speechbrain/lobes/models/transformer/Transformer.py

```diff
@@ -420,6 +422,8 @@ def __init__(
             ]
         )
         self.norm = sb.nnet.normalization.LayerNorm(d_model, eps=1e-6)
+        self.layerdrop_prob = layerdrop_prob
+        self.rng = np.random.default_rng(0)  # for DDP so same layer(s) are dropped
```

Comment:
can't we use a given seed instead? Problem is that the same layer will ALWAYS be dropped -> find a way to use self._seed instead ?

Reply:
Does it really make sense to have a seed variable? It doesn't seem to me like one would ever want to change it (so a constant seems fine to me).